IMPLEMENTATION_PLAN_bgstream_webui_auth.md

```
@@ -0,0 +1,19 @@
+# Implementation Plan: bgstream_webui_auth
+
+## Stage 1: Reproduce Stream Auth Failure
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)